### PR TITLE
Add gfo-light.owl

### PR DESCRIPTION
**Pull request by AlexU75**

### CODE REVIEW

**Comment by ChristophB** [comment]
Could you please add a short description of `gfo-light.owl` to `README.md`.

**Review by ChristophB — APPROVED** [review]

**Comment by ChristophB** [comment]
Please use "Squash and merge" to merge this pull request!

**Review by RalphBln — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Work on GFO has started in 1999 in the context of the GOL project (General Ontol
 The "native" formalization language for GFO is [first-order logic (FOL)](https://en.wikipedia.org/wiki/First-order_logic). Partial axiomatizations of GFO in FOL exist in report working drafts, but are not yet available to the public.
 
 ### OWL
-There is an [OWL](http://en.wikipedia.org/wiki/Web_Ontology_Language) version of GFO, which currently comprises a stable core called *gfo-basic.owl* and a more extensive version called *gfo.owl*.
+There is an [OWL](http://en.wikipedia.org/wiki/Web_Ontology_Language) version of GFO, which currently comprises a stable core called *gfo-basic.owl* and a more extensive version called *gfo.owl*. *gfo-light.owl* contains basic concepts and axioms described in the current GFO papers and is primarily intended to found domain or application ontologies.
```
Comment (k00ni) [inline README.md:21]: Why keeping the reference to gfo-basic.owl at this point? Isn't gfo-light.owl meant to replace it?

Comment (AlexU75) [reply README.md]: I have described the distinction between gfo-basic and gfo-light in the README.

File: README.md [hunk 2]
```diff
@@ -17,11 +17,8 @@ Work on GFO has started in 1999 in the context of the GOL project (General Ontol
...
+*gfo-basic.owl* forms the core module of GFO. We are going to continually provide further modules/extensions, dedicated to several of our research domains, like time, space, processes, etc. There may be parallel extensions for these domains, and not all extensions will be consistent with each other. All module files are provided with their own namespaces (e.g., http://purl.org/ontology/gfo-time) and can be used separately. A selection of those modules will be unified in the next version of *gfo.owl* (under the namespace http://purl.org/ontology/gfo). *gfo-light.owl* is a simplified view of the entire GFO. It does not build on single modules but combines all basic concepts and simplified axioms of the GFO. *gfo-light.owl* is primarily intended to found domain or application ontologies.
```
Comment (k00ni) [inline README.md:21]:
```suggestion
*gfo-light.owl* forms the core module of GFO. We are going to continually provide further modules/extensions, dedicated to several of our research domains, like time, space, processes, etc. There may be parallel extensions for these domains, and not all extensions will be consistent with each other. All module files are provided with their own namespaces (e.g., http://purl.org/ontology/gfo-time) and can be used separately. A selection of those modules will be unified in the next version of *gfo.owl* (under the namespace http://purl.org/ontology/gfo). *gfo-light.owl* is a simplified view of the entire GFO. It does not build on single modules but combines all basic concepts and simplified axioms of the GFO. *gfo-light.owl* is primarily intended to found domain or application ontologies.
```

**Comment by k00ni** [comment]
Out of curiosity: why did you introduce a new file (gfo-light) and not update the existing one (gfo-basic)?

**Comment by AlexU75** [comment]
Because the two ontologies are different. gfo-basic represents only one module (the core module) of the GFO, while gfo-light is a simplified view of all/multiple modules.

**Comment by ChristophB** [comment]
Since this pull request is already a year old, I will now merge it into main.

**Pull request merged**